nim/tests/test_routes.py

```diff
@@ -0,0 +1,110 @@
+import json
+from nim.app import app
+import pytest
+
+
+@pytest.fixture
+def client_app():
+    return app.test_client()
+
+
+def assert_new_game_success(passed_client, req):
```

Comment:
`passed_`? That sounds strange, of course the argument was passed.
Maybe just `client`?

There may be something better than `req` too. `game_data`?

Reply:
You seem to have missed this one.